### ISSUE TITLE
Icon Request: fa-old-tv

### ISSUE THREAD

**Issue by tuxy** [FortAwesome/Font-Awesome#7144]
Thank you for adding long requested television icon in version 4.4 but it looks more like a modern computer monitor then televison. I highly recommend renaming the new fa-television icon to something like ``fa-hdtv`` and making a new old style television under ``fa-tv`` that looks something like this bellow;

[!TV](http://i.imgur.com/kcehoAy.png)

Thank You!

**Comment by albino1** [comment]
+1 

This is a much more universal method of indicating television than the current one which looks like a computer monitor.

**Comment by tuxy** [comment]
@albino1 I agree, the one that was added looks more like HDTV or LCD then TV.

**Comment by albino1** [comment]
Yep.  I understand that the one added recently actually looks much more like a realistic modern TV, but for the purposes of describing "television" or "the act of watching television" you really need something more like the image you posted above.

**Comment by nielhardt** [comment]
+1

**Comment by tuxy** [comment]
+1 @albino1

**Comment by ggerke** [comment]
+1 as well... although I like the example TV shown above I think I'd prefer the antennas to be the same length

**Comment by leahss13** [comment]
+1 I want this soon...

**Comment by aethior** [comment]
Hi @ghost,

This icon was added in v5.0.0.
Check https://fontawesome.com/icons/tv-retro

Regards.

cc/ @tagliala
